docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        spsa = NELDER_MEAD(80, callback=callback)
</pre><p>Is Nelder Mead a kind of SPSA optimizer? My understanding form your review paper is that this is not the case. Can you change the variable name <code>spsa</code> to <code>optimizer</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Fixed